Emitron/Emitron/UI/Empty States/NoResultsView.swift

```diff
@@ -28,11 +28,23 @@
 
 import SwiftUI
 
-struct NoResultsView<Header: View>: View {
-  @EnvironmentObject var tabViewModel: TabViewModel
-  var contentScreen: ContentScreen
-  let header: Header
+struct NoResultsView<Header: View> {
+  init(
```

Comment:
If you don't do that, the properties don't get to be private. I always start out `private` and only increase access as necessary. Otherwise it's not clear from looking at a type how much of it might need to be used from any other file.

Reply:
__That__ is precisely what I was missing. Thanks—I'd forgotten that the autogenerated initialisers don't work with private properties.